user-guide/Reference/Metrics/swarming_elements_benchmarks.md

```diff
@@ -0,0 +1,27 @@
+---
+uid: swarming_bookings_benchmarks
+---
+
+# Swarming bookings benchmarks
+
+## Benchmarks
+
+| \# | Specification | Scope | Metric | Test Setup |
+| -- | ------------- | ----- | ------ | -------- |
+| 1 | Swarming 1 element | DMS | 0.591 s | DaaS cluster. |
+| 2 | Swarming 10 elements | DMS | 1.590  s | DaaS cluster. |
+| 3 | Swarming 100 elements | DMS | 9.670 s | DaaS cluster. |
+| 4 | Raw swarming time for 1 element | DMS | 0.154 s | DaaS cluster.<br /> Measured swarm action only, without time needed to start the element. |
+| 5 | Failover switch 100 elements | DMS | 50 s | Non-swarming system. Measured until Cube availability. |
+| 6 | Swarming 100 elements (Failover-like switchover) | DMS | 10 s | First element available after 2 seconds. |
+| 7 | Agent startup with 100 elements present | DMS | 6.353 s | Loading from DB. |
```

Comment:
local dma with remote staas so different from daas+staas or local+cc.

Perhaps we add this info? or redo the timings for the other setups

Reply:
Best to add what was used for this specific measurement